aries-site/src/examples/components/layer/LayerCenterExample.js

```diff
@@ -25,18 +33,21 @@ export const LayerCenterExample = () => {
                 <Box justify="center">
                   <MailOption />
                 </Box>
-                <Heading margin="none" level={2}>
+                <Heading margin="none" level={4}>
                   Modal Dialog
                 </Heading>
               </Box>
               <Box justify="center">
                 <Button icon={<FormClose />} onClick={onClose} />
               </Box>
             </Box>
+            <Paragraph margin="none">A subtitle if needed</Paragraph>
             <Box overflow="auto" pad={{ vertical: 'medium' }}>
               <Text>
-                For modal dialogs, the use case will determine the design and
-                size of the box for your content
+                The width of this modal dialog is 384px with 12px corner radius.
```

Comment:
I am inclined to reference the width and corner radius in tshirt sizes, with px as supporting if we need it. Trying to reinforce language of tshirt sizes
```suggestion
                The width of this modal dialog is 'medium' (384px) with 'small' (12px) corner radius.
```